common/src/autogluon/common/savers/save_pkl.py

```diff
@@ -17,13 +22,16 @@ def save(path, object, format=None, verbose=True, **kwargs):
     else:
         raise ValueError(f'compression_fn={compression_fn} is not a valid compression_fn. Valid values: {compression_fn_map.keys()}')
 
-    pickle_fn = lambda o, buffer: pickle.dump(o, buffer, protocol=4)
+    def pickle_fn(o, buffer):
+        return pickle.dump(o, buffer, protocol=4)
```

Comment:
Is this a recommendation provided by something to change? (Mostly my own curiosity)

Reply:
It's a flake recommendation: https://www.flake8rules.com/rules/E731.html